### PR TITLE
created tags page and added new pic

### PR DESCRIPTION
**Pull request by gmammadova**
Created the tags page and image with attribution.

### CODE REVIEW

**Comment by gmammadova** [comment]
cc @elvinefendi

File: tags.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (ElvinEfendi) [inline tags.html]: redundant whitespace

File: tags.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  <head>
+    <title>HTML Tags</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <h1>Important HTML tags</h1>
+
+    <a href="https://www.flickr.com/photos/jdhancock/3814523970">
+
+    <img src="images/storm_trooper_tagged.jpg" alt="Tagged Storm Trooper">
```
Comment (ElvinEfendi) [inline tags.html]: this is within <a> tag, and therefore it should be indented

**Review by ElvinEfendi — COMMENTED** [review]
Just two comments, otherwise LGTM

**Review by ElvinEfendi — APPROVED** [review]
💯 

pull

**Pull request merged**